release-notes/README.md

```diff
@@ -4,6 +4,22 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## December 22, 2021
+
+### iOS AEPCore 3.3.2
+
+* A fix to Identity.getIdentifiers API
+* A fix to configureWithAppId API
+* An update to configureWithAppId to support removing appID from persistence when passing empty app id
```

Comment:
Added support to configureWithAppID for removing AppID from persistence when passing an empty app ID.